src/server/main.py

```diff
@@ -9,6 +9,8 @@
 from server.config import CONFIG
 
 
+STATIC_URL = "https://storage.googleapis.com/static-covid/static/staging"
```

Comment:
I would initialize this from the env variable directly, so:

```
STATIC_URL = os.getenv("STATIC_URL", "/static")
```

Reply:
And then:
1. add to https://github.com/epidemics/covid/blob/master/deploy/chart/values.staging.yaml (similarly for values.production.yaml):
```
env:
  staticUrl: "https://storage.googleapis.com/static-covid/static/staging"
```
2. to https://github.com/epidemics/covid/blob/master/deploy/chart/templates/configmap.yaml#L6 add
```
  STATIC_URL: {{ .Values.env.staticUrl | quote }}
```

Reply:
I think that would all make it work out of the box with docker-compose/locally and would still fetch from gcs in deployments.

Also, please remove ENABLE_NOTION env, we don't use it for anything anymore.

Reply:
ok

Reply:
Yes, I could confirm that local docker is able to use GCS static files. Used it this way.